content/play/download.md

```diff
@@ -1,46 +1,65 @@
 ---
 title: "Download Lc0"
 weight: 200
+# layout: download
 ---
 
 The latest stable version of Lc0 is **[{{< param lc0version >}}](https://github.com/LeelaChessZero/lc0/releases/tag/{{<param lc0version>}})**.
```

Comment:
Maybe change this too to be one "`Latest stable version: v0.31.2 | All releases`" line like in http://pr134.drafts.lczero.org/play/quickstart/ ?